examples/apis/utils/ssh.py

```diff
@@ -0,0 +1,25 @@
+from avocado.utils.ssh import Session
+from avocado.utils.process import SubProcess
+import time
+
+with Session('host', user='root', key='/path/to/key') as s:
+    print('connected')
```

Comment:
why not logger, using logger will be good

Reply:
This is just an example script, so I tried to keep it as simple as possible.